### DISCUSSION TITLE
release 2.3.0

### DISCUSSION THREAD

**Discussion by Aloren** [aerospike/spring-data-aerospike#521] [Team Posts]
Hello @rbotzer
We have a new version of `spring-data-aerospike`  ready for release `2.3.0.RELEASE`.

I've created release draft: https://github.com/aerospike-community/spring-data-aerospike/releases/tag/untagged-22d42e36cd292467a31a
and updated version in master.

Could you please release the version?

**Comment by Aloren** [comment]
@arrowplum Probably you could help us with release?

**Comment by rbotzer** [comment]
@Aloren sorry about the late reply - I was away with little connectivity, however Brian Nichols had built and uploaded the latest version: https://search.maven.org/search?q=a:spring-data-aerospike

BTW, what's going on with release 2.3.1? I see in the tags/releases that it exists.

**Comment by Aloren** [comment]
@rbotzer Thanks, I've already seen and we are already using it! 👍 
Release 2.3.1 is going to have mostly changes in reactive operations related to index. @kptfh is going to work on this, he is currently on vacation, so I decided to move those changes to the separate release instead of 2.3.0 as they are not finished yet.
